src/test/integration/scheduler/test_unpack_only.py

```diff
@@ -1,37 +1,22 @@
-# pylint: disable=wrong-import-order,attribute-defined-outside-init
-import gc
-from multiprocessing import Queue
+import pytest
 
 from objects.firmware import Firmware
-from scheduler.unpacking_scheduler import UnpackingScheduler
-from storage.unpacking_locks import UnpackingLockManager
 from test.common_helper import get_test_data_dir
-from test.integration.common import MockFSOrganizer
+from test.integration.conftest import SchedulerIntegrationTestConfig
 
 
 class TestFileAddition:
-    def setup(self):
-        self._tmp_queue = Queue()
-        self.unpacking_lock_manager = UnpackingLockManager()
-        self._unpack_scheduler = UnpackingScheduler(
-            post_unpack=self._dummy_callback,
-            fs_organizer=MockFSOrganizer(),
-            unpacking_locks=self.unpacking_lock_manager,
-        )
-        self._unpack_scheduler.start()
-
-    def teardown(self):
-        self._unpack_scheduler.shutdown()
-        self.unpacking_lock_manager.shutdown()
-        self._tmp_queue.close()
-        gc.collect()
-
-    def test_unpack_only(self):
+    @pytest.mark.SchedulerTestConfig(
+        SchedulerIntegrationTestConfig(
+            start_processes=True,
```

Comment:
`start_processes: bool = True` seems to be the default in `SchedulerTestConfig`, so why do we need to set this?

Reply:
Removed.